eslint.config.js

```diff
@@ -0,0 +1,31 @@
+module.exports = [
```

Comment:
Is this replacing the existing `.eslintrc.js`, if so lets remove that file?

Reply:
It is.  I was only leaving the other one in case we needed to go back, but I suppose, git, so it'll be there.

Reply:
Oh okay. It should've still worked with the old file, but they just made this format be the default.
Why was `'extends': 'eslint:recommended',` removed?

Reply:
Ok, added per https://eslint.org/docs/latest/use/migrate-to-9.0.0#-eslintrecommended-and-eslintall-no-longer-accepted-in-flat-config but a little different since we can't use `import` here with our setup.

I confirmed the difference in configs with and without the recommended rules by printing config for a file:
```
yarn eslint --print-config "assets/javascripts/cable.js"
```

Reply:
Thank you!